recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2023.7.22" %}
+{% set version = "2023.11.17" %}
 
 {% set pip_version = "23.2.1" %}
 {% set setuptools_version = "68.2.2" %}
```

Comment:
It's not needed. pip/setuptools depend on certify, so to avoid a circular dependency we use those to bootstrap the build. We'll only have to update these versions when they become incompatible with our latest python target.